tests/proxy_spec.lua

```diff
@@ -531,4 +531,71 @@ describe("Monitored proxy objects", function ()
                 assert.equals(params.proxy, proxy)
                 assert.is_true(params.appeared)
            end)
+
+        it("can call async methods", function ()
```

Comment:
Thanks for adding tests to the monitored proxy objects. Can you add tests for the non-monitored proxies too please (they start around line 188)? I think you can just add a couple of cases to existing tests.

Reply:
Sorry, I didn't actually notice the tests were separated like this. The test itself is using the non-monitored proxy initialized in the beginning of the `describe` block. The method calls seem not to differ between monitored and non-monitored proxies, should maybe I just move the test to the monitored proxy section?

Reply:
Please keep them both if you don't mind. The tests on `_proxy` should be simpler.